lib/Frontend/CompilerInvocation.cpp

```diff
@@ -3254,7 +3254,7 @@ static bool ParseIRGenArgs(IRGenOptions &Opts, ArgList &Args,
   if (auto A = Args.getLastArg(OPT_enable_round_trip_debug_types,
                                OPT_disable_round_trip_debug_types)) {
     Opts.DisableRoundTripDebugTypes =
-        Args.hasArg(OPT_disable_round_trip_debug_types);
+        A->getOption().matches(OPT_disable_round_trip_debug_types);
   }
```

Comment:
Not NFC. cc @slavapestov

Reply:
Why not just remove the `auto A = ` and keep this line as-is?

Reply:
My impression is that we want to know what the last occurrence of either of these two flags is. 
`-enable -disable -enable` amounts to "enable", right?

Reply:
LGTM